### ISSUE TITLE
Improve site link handling

### ISSUE THREAD

**Issue by kaso17** [Jackett/Jackett#1473]
Currently the default URL will be used on early requests (e.g. catpchas, etc.). This sometimes will prevent users from configuring some indexers (e.g. if the default URL is blocked by the ISP).

We should always let users change the URL.

A workaround for this is to manually create the indexer json config file and setting the corresponding sitelink there.

**Comment by Linwelin** [comment]
I have created a json file and added this, but it is not showing up in jackett?

[
  {
    "id": "alternatelink",
    "type": "inputstring",
    "name": "Alternate Link",
    "value": "http://kong.net-freaks.com"
  },
]

**Comment by kaso17** [comment]
Try
```json
[
  {
    "id": "sitelink",
    "type": "inputstring",
    "name": "Site Link",
    "value": "http://kong.net-freaks.com/"
  },
]
```
Filename must be iptorrents.json
If you just copy and rename an existing file and adjust the sitelink it should work.

**Comment by Linwelin** [comment]
tried the one you posted, the indexer iptorrents still not showing. I copied one that was already added just changed the name and link, still nothing :(

**Comment by Linwelin** [comment]
it's working now restarted PC and now it shows :D Thank you kaso17

**Comment by Dallustallus** [comment]
Hi,

Sorry to be a pain, but where are we suppose to save the iptorrents.json file under Windows? I don't even see any existing .json

Definitions folder has .yml and IPT is not even there.

Thanks in advance

**Comment by Dallustallus** [comment]
Thank you so much. Just tested and it works like a charm.

I have been searching everywhere as to where to place the .json file.

Thank you

**Comment by Anshuashmeet** [comment]
Hi,

Where can I place the file on my Synology NAS?

Thanks

**Comment by kaso17** [comment]
@Anshuashmeet I belive it was /usr/local/jackett/var/.config/Jackett/

**Comment by gordymuir** [comment]
Hi guys 
Done all of the above and got to the json file edited in notepad but when I click save I get access denied. 
What have I missed along the way?

**Comment by mcmurray** [comment]
This isnt working for me even when i use a new working link 

https://ip.findnemo.net

Any ideas?

**Comment by ngosang** [comment]
I think this issue was fixed in #7104 The latest version of Jackett already included this fix.
I'm going to close, if someone has problems I will reopen.
